flink-java/src/test/java/org/apache/flink/api/java/operator/JoinOperatorTest.java

```diff
@@ -1211,7 +1210,6 @@ public String toString() {
 		public String myString;
 		public Object nothing;
 		public List<String> countries;
-		public Writable interfaceTest;
```

Comment:
The interface here checks for correct type extraction and handling of Flink. For that this variable does not need to be used but just declared. 
However, this functionality is tested at other places as well. 
So, +1 for removing this line.